libs/geometry/include/geometry/SurfaceOrientation.h

```diff
@@ -47,67 +47,59 @@ class SurfaceOrientation {
     public:
         Builder();
         ~Builder() noexcept;
+        Builder(Builder&&) = default;
```

Comment:
Unfortunately you can't do that because this can lead to a double free. You need to std::swap() the impl pointer in the operator== implementation. The the move ctor implementation you can do better and just set the "rhs" pointer to nullptr.

Reply:
Done.